DependencyInjection/TrikoderOAuth2Extension.php

```diff
@@ -64,7 +65,7 @@ private function configureAuthorizationServer(ContainerBuilder $container, array
     {
         $authorizationServer = $container
             ->getDefinition('league.oauth2.server.authorization_server')
-            ->replaceArgument('$privateKey', $config['private_key'])
+            ->replaceArgument('$privateKey', new CryptKey($config['private_key'], null, false))
```

Comment:
Since we're not checking for file permissions anymore, the `chmod` workaround from `Tests/TestKernel.php` in the `boot` method can be removed.

Reply:
Are you sure to remove the ```boot``` method at all, since it sets environment variables there?
```php   
    public function boot()
    {
        putenv(sprintf('PRIVATE_KEY_PATH=%s', TestHelper::PRIVATE_KEY_PATH));
        putenv(sprintf('PUBLIC_KEY_PATH=%s', TestHelper::PUBLIC_KEY_PATH));
        putenv(sprintf('ENCRYPTION_KEY=%s', TestHelper::ENCRYPTION_KEY));
    }
```
I'll leave it there for the moment, o.k.?

Reply:
Yeah, I only meant to remove the `chmod` calls so it looks like this:

```php
/**
 * {@inheritdoc}
 */
public function boot()
{
    putenv(sprintf('PRIVATE_KEY_PATH=%s', TestHelper::PRIVATE_KEY_PATH));
    putenv(sprintf('PUBLIC_KEY_PATH=%s', TestHelper::PUBLIC_KEY_PATH));
    putenv(sprintf('ENCRYPTION_KEY=%s', TestHelper::ENCRYPTION_KEY));

    parent::boot();
}
```

Reply:
Also done... sorry, dropped ```parent::boot()``` accidentally in comment. Thanks a lot for your help and input!